apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -57,7 +72,7 @@ window.PLOrderBlocks = function (uuid, options) {
       const dropzoneBlocks = Array.from(
         $(dropzoneElementId)[0].querySelectorAll('.pl-order-block'),
       );
-      if (!block.classList.contains('pl-order-blocks-selected')) {
+      if (block.getAttribute('aria-selected') === 'false') {
```

Comment:
check out the HTML spec for boolean attributes here: https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML 

Technically, if the attribute exists at all, it should be interpreted as true. So when 'selected' should be false, just remove the attribute entirely. When it should be true, add the attribute back. Don't have any value for the attribute.